src/Domain/Tags.hpp

```diff
@@ -289,7 +301,7 @@ struct InterfaceImpl<true, DirectionsTag, NameTag, FunctionTag>
 
 template <typename DirectionsTag, typename NameTag, typename FunctionTag>
 struct InterfaceBase
-    : db::PrefixTag,
+    : virtual db::PrefixTag,
       Interface_detail::InterfaceImpl<db::is_compute_item_v<FunctionTag>,
                                       DirectionsTag, NameTag, FunctionTag> {
   static std::string name() noexcept { return "Interface"; }
```

Comment:
This is not related to the PR (I thought of this will looking at the tags in the tests), but should the name here not include the tag being wrapped or something? Whenever that gets changed (an issue and a later PR is totally fine with me) we should make sure to have a test for it too :)

Reply:
This is true of every prefix tag in the code.  Someone should just fix them all instead of filing a bug for each one individually whenever someone happens to think of it.